### PR TITLE
add draft feb 2022 newsletter

### PR DESCRIPTION
**Pull request by davidwyld**
In addition to reviewing the newsletter, we should decide whether this is to be rebranded as the :sparkles: **TUOS RSE Community Newsletter** :sparkles: - but I will leave that for after the community meeting.

### CODE REVIEW

File: _newsletters/2022-02-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+**Previously scheduled event cancelled, watch this space**
+* **Parallelisation: an easy trick to speed up your code?**
+Parallelisation - running multiple pieces of work at the same time - can be a great way to speed up research code, but requires an appreciation of the costs introduced by distributing work concurrently over computational assets. This session will be made up of short talks introducing both fundamental concepts needed for successful use of parallelisation, and state of the art technologies available for researchers.
+
+More details about the event are available [on our website](https://rse.shef.ac.uk/events/lunchbytes-2021-12-02.html).
+
+
+#### LunchBytes needs YOU!
+LucnBytes are organised by and for the research software community at The University of Sheffield. If you'd like to curate a session on a topic or present someting, get in touch by emailing [lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) - Or suggest topics [on the jamboard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: "LucnBytes"

File: _newsletters/2022-02-10-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+#### Semantic Release
+[**semantic-release**](https://github.com/semantic-release/semantic-release) automates the whole package release workflow including: determining the next version number, generating the release notes, and publishing the package.
+
+This removes the immediate connection between human emotions and version numbers, strictly following the Semantic Versioning specification and communicating the impact of changes to consumers.
+
+
+### Events
+
+* [UK National GPU Hackathon 2022, 28th February and 7-9th March](https://www.gpuhackathons.org/event/uk-national-gpu-hackathon-2022) - There's still time to register for the 2022 GPU Hackathon. Team up with experienced GPU mentors to learn and apply accelerated and parallel computing to your project. **Application deadline is on 10th of January.**
```
Comment (ptheywood) [inline _newsletters/2022-02-10-newsletter.md]: This is past the deadline, so either wants amending to reflect that or remove?

File: _newsletters/2022-02-10-newsletter.md (hunk 1 above)
Comment (Robadob) [reply _newsletters/2022-02-10-newsletter.md]: "someting"

**Review by ptheywood — CHANGES_REQUESTED** [review]

File: _newsletters/2022-02-10-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+More details about the event are available [on our website](https://rse.shef.ac.uk/events/lunchbytes-2021-12-02.html).
+
+
+#### LunchBytes needs YOU!
+LucnBytes are organised by and for the research software community at The University of Sheffield. If you'd like to curate a session on a topic or present someting, get in touch by emailing [lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) - Or suggest topics [on the jamboard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/)
+
+
+### News
+
+* [Software Sustainability  Institute Fellows 2022 Announced](https://www.software.ac.uk/news/announcing-2022-software-sustainability-institute-fellows) - Among this year's cohort is Sheffield RSE Team member, Dr David Wilby. During his fellowship, David will be developing guidance and training resources around best practices for sustainable research software development in MATLAB, watch this space!
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: Double space between "Software Sustainability" and "Institute" (markdown processing may strip this anyway)

File: _newsletters/2022-02-10-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+  * We invite authors who would like feedback on the reproducibility of computationally intensive research to submit details of their paper, code and data for reproduction and review. You can submit your paper on the ReproHack Hub. To ensure your paper is associated with this event, please make sure to select the event on the form during submission. If you would like to make your work available for future HPC ReproHacks, we recommend including an HPC tag. Please see our [Author Guidelines](https://www.reprohack.org/author_guidelines) for more information.
+
+* [French government has awarded Open Science prizes for research software](https://www.enseignementsup-recherche.gouv.fr/fr/remise-des-prix-science-ouverte-du-logiciel-libre-de-la-recherche-83576) (en Français)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: 1 line summary/teaser?

File: _newsletters/2022-02-10-newsletter.md (hunk 2 above)
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: Application date has passed.

File: _newsletters/2022-02-10-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+
+### Opportunities
+
+* [PhD in GPU computing at CERN - see link for details](https://docs.google.com/document/d/1tq21-XzqbrG_nMpWDxisXLAAzIeE0AQ46Y5mE5othPI)
+
+* [Research Data Scientist/Research Software Engineer @ The Alan Turing Institute](https://cezanneondemand.intervieweb.it/turing/jobs/research_data_scientistresearch_software_engineer_20259/en/)
+
+* [RSE @ Cardiff](https://krb-sjobs.brassring.com/TGnewUI/Search/Home/HomeWithPreLoad?partnerid=30011&siteid=5460&PageType=searchResults&SearchType=linkquery&LinkID=6#jobDetails=1919917_5460)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: 18th Feb deadline, perhaps highlight short notice.

File: _newsletters/2022-02-10-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+* [Research IT Infrastructure Analyst @ Manchester](https://www.jobs.manchester.ac.uk/displayjob.aspx?jobid=21611)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: 14th Feb deadline.

File: _newsletters/2022-02-10-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+* [HPC Solution Area Specialist @ Microsoft](https://careers.microsoft.com/us/en/job/1239668/EMEA-GBB-HPC-Solution-Area-Specialist)
+
+* [Senior RSE @ Imperial](https://www.imperial.ac.uk/jobs/description/FOG00559/senior-research-software-engineer)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: 21st feb

File: _newsletters/2022-02-10-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+* [Head of RSE @ Exeter](https://jobs.exeter.ac.uk/hrpr_webrecruitment/wrd/run/ETREC107GF.open?VACANCY_ID=307457YUr1&WVID=3817591jNg&LANG=USA)
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md]: 23rd feb

File: _newsletters/2022-02-10-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+  * [10th March 10:00-16:30](https://rse.shef.ac.uk/training/workshop/2022-03-10-git-zero-hero)
+  * [5th April 10:00-16:30](https://rse.shef.ac.uk/training/workshop/2022-04-05-git-zero-hero)
+
+
+* 12-week Parallel Computing with GPUs Course:
+  * The course has started but late registrations are available until week 4 when the GPU content starts. The course has been designed as an undergraduate 4th year (and MSc) module for Computer Science but is available to staff and PhD Students (as part of the DDP program) and regularly has staff and PhD student enrolment. The first 3 weeks are focused on teaching C and OpenMP followed by GPU programming with CUDA C. Lectures are provided as pre-recorded mini lectures and there are 2 hours of scheduled support per week to undertake practical lab classes. Assessment is not required for DDP or staff participants. If you are interested in enrolling as staff or a PhD student then please contact the module leader [p.richmond@sheffield.ac.uk](mailto: p.richmond@sheffield.ac.uk)
+
+
+### Opportunities
```
Comment (Robadob) [inline _newsletters/2022-02-10-newsletter.md:115]: Could be good to highlight all the deadlines (though the Microsoft one doesn't appear to have one publicly listed), and perhaps order by deadline.

**Review by Robadob — APPROVED** [review]
Bunch of changes suggested, some necessary (e.g. typos, passed deadlines), most optional.

**Pull request merged**